src/main/java/bowling/view/InputView.java

```diff
@@ -0,0 +1,27 @@
+package bowling.view;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class InputView {
+
+    private static final Scanner SCANNER;
+
+    static {
+        SCANNER = new Scanner(System.in);
+    }
```

Comment:
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```
굳이 static 블럭을 쓸필요 없어보이는데 사용하신 이유가 있으신가요? 🤔